airbyte-integrations/bases/base-normalization/Dockerfile

```diff
@@ -1,23 +1,26 @@
-FROM airbyte/base-airbyte-protocol-python:0.1.1
-
-# dbt needs git in order to run.
-RUN apt-get update && apt-get install -y git && rm -rf /var/lib/apt/lists/*
+FROM fishtownanalytics/dbt:0.19.1
+COPY --from=airbyte/base-airbyte-protocol-python:0.1.1 /airbyte /airbyte
 
 WORKDIR /airbyte
 COPY entrypoint.sh .
 
 WORKDIR /airbyte/normalization_code
 COPY normalization ./normalization
 COPY setup.py .
-RUN pip install .
 COPY dbt-project-template/ ./dbt-template/
+
+WORKDIR /airbyte/base_python_structs
```

Comment:
where is does this directory correspond to? can't seem to find it in the monorepo. just trying to understand the general flow here.

Reply:
It's in `airbyte-integrations/bases/airbyte-protocol/Dockerfile`. I think this is temporary since we'll be using the pypi airbyte-cdk soon?

Reply:
the directory is created in the docker image from airbyte-protocol in order to install the airbyte protocol python files.

So yes, once it is on pypi, we can just replace this by a new line in the setup.py/requirements.txt instead